debian/focal/foreman/rules

```diff
@@ -7,31 +7,32 @@
 
 DEB_BUILD_OPTIONS=noautodbgsym
 
+RUBY_VERSION := $(shell ruby -e 'print RbConfig::CONFIG["RUBY_API_VERSION"]')
+BUNDLE := $(firstword $(wildcard /usr/bin/bundle$(RUBY_VERSION) /usr/bin/bundle))
```

Comment:
I wonder if we should apply the same changes to the other `rules` files, even if it'll pick up `/usr/bin/bundle` there.

I'd probably also keep the (now useless) `gem fetch thor` here, just for the sake to keep all `rules` files identical. Makes edits so much easier.

What do you think?

Reply:
If it's safe and keeps our scripts the same, I can see the benefit for that.

Reply:
Yeah, there is no `/usr/bin/bundle2.5` on Buster and Bionic, so it will fall-through to `/usr/bin/bundle` and keep doing what it does today.

Reply:
Looking ahead at Bullseye, it looks like there is `/usr/bin/bundle2.7`. Would that behave like focal?

Reply:
Yes. Ideally, `cp -a focal bullseye` and done.

Reply:
Speaking of which, I think you looked into merging them. Right now it looks like they are all the same except for a changelog entry in `foreman-installer`. Are you still looking into merging them?
```console
$ diff -Nutr bionic/ buster/
diff -Nutr bionic/foreman-installer/changelog buster/foreman-installer/changelog
--- bionic/foreman-installer/changelog	2021-09-06 12:15:00.682999880 +0200
+++ buster/foreman-installer/changelog	2021-09-06 12:15:00.960003684 +0200
@@ -14,7 +14,7 @@
 
   * Require puppet 6.15.0+
 
- -- Eric D. Helms <ericdhelms@gmail.com>  Wed, 19 May 2021 11:58:40 -0400
+ -- Eric D. Helms <ericdhelms@gmail.com>  Wed, 19 May 2021 11:58:45 -0400
 
 foreman-installer (2.6.0-1) stable; urgency=low
 
$ diff -Nutr bionic/ focal/
diff -Nutr bionic/foreman-installer/changelog focal/foreman-installer/changelog
--- bionic/foreman-installer/changelog	2021-09-06 12:15:00.682999880 +0200
+++ focal/foreman-installer/changelog	2021-09-06 12:15:01.229007377 +0200
@@ -14,7 +14,7 @@
 
   * Require puppet 6.15.0+
 
- -- Eric D. Helms <ericdhelms@gmail.com>  Wed, 19 May 2021 11:58:40 -0400
+ -- Eric D. Helms <ericdhelms@gmail.com>  Wed, 19 May 2021 11:58:18 -0400
 
 foreman-installer (2.6.0-1) stable; urgency=low
 
```

Reply:
Yes, still my plan, even if the current PR actually makes the `control` file diverge (for now).

Reply:
To me that would be a good reason to copy the bundle changes into bionic & buster.

Reply:
I did copy over the `debian/rules` changes, for `debian/control`, that would break the builds, as `d/control` on Focal now drops the `bundler` dependency. On the other hand, we don't *need* to drop that dep... mhhh :thinking: